README.md

```diff
@@ -0,0 +1,147 @@
+# otel-desktop-viewer
+
+`otel-desktop-viewer` is a CLI tool for receiving OpenTelemetry traces while working
+on your local machine that helps you visualize and explore your trace data without
+needing to send it on to a telemetry vendor.
+
+![OpenTelemetryDesktopViewer](https://user-images.githubusercontent.com/56372758/217080670-3001cb67-ab20-4ae2-ac55-82ca04bad815.png)
+
+Its goals are to be easy-to-install with minimal dependencies and fast. It is written in Go
+as a custom exporter on top of the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector).
+
+## Getting started
+
+Make sure you have [go](https://go.dev/) installed.
+
+```bash
+# install the CLI tool
+go install github.com/CtrlSpice/otel-desktop-viewer@v0.1.0
```

Comment:
We haven't tagged a version yet, so we should do that soon. Right now you would have to run `go install github.com/CtrlSpice/otel-desktop-viewer@latest`